build.gradle

```diff
@@ -6,89 +6,77 @@ plugins {
-        }
-        maven {
-            name 'Bintray'
-            url 'https://dl.bintray.com/exactpro/sailfish'
-        }
-        maven {
-            name 'Artifactory-snapshot-oss'
-            url 'https://oss.jfrog.org/artifactory/libs-snapshot'
-        }
-        mavenLocal()
+    maven {
+        name 'Bintray'
+        url 'https://dl.bintray.com/exactpro/th2'
+    }
+    maven {
+        name 'Bintray'
+        url 'https://dl.bintray.com/exactpro/sailfish'
+    }
+    maven {
+        name 'Artifactory-snapshot-oss'
+        url 'https://oss.jfrog.org/artifactory/libs-snapshot'
+    }
+    mavenLocal()
 
 
-        configurations.all {
-            resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
-            resolutionStrategy.cacheDynamicVersionsFor 0, 'seconds'
-        }
+    configurations.all {
+        resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
+        resolutionStrategy.cacheDynamicVersionsFor 0, 'seconds'
     }
+}
 
-    dependencies {
-        api platform('com.exactpro.th2:bom:2.10.1')
+jar {
+    manifest {
+        attributes(
+                'Created-By':               "${System.getProperty('java.version')} (${System.getProperty('java.vendor')})",
+                'Specification-Title':      '',
+                'Specification-Vendor':     'Exactpro Systems LLC',
+                'Implementation-Title':     project.archivesBaseName,
+                'Implementation-Vendor':    'Exactpro Systems LLC',
+                'Implementation-Vendor-Id': 'com.exactpro',
+                'Implementation-Version':   project.version
+        )
     }
+}
 
-
-    jar {
-        manifest {
-            attributes(
-                    'Created-By':               "${System.getProperty('java.version')} (${System.getProperty('java.vendor')})",
-                    'Specification-Title':      '',
-                    'Specification-Vendor':     'Exactpro Systems LLC',
-                    'Implementation-Title':     project.archivesBaseName,
-                    'Implementation-Vendor':    'Exactpro Systems LLC',
-                    'Implementation-Vendor-Id': 'com.exactpro',
-                    'Implementation-Version':   project.version
-            )
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            from components.java
         }
     }
 
-    publishing {
-        publications {
-            mavenJava(MavenPublication) {
-                from components.java
-            }
-        }
-
-        repositories {
-            maven {
-                name = 'localRepo'
-                url = sharedDir
-            }
+    repositories {
+        maven {
+            name = 'localRepo'
+            url = sharedDir
         }
     }
-
-}
-
-apply plugin: 'kotlin'
-apply plugin: 'application'
-apply plugin: 'com.palantir.docker'
-apply plugin: 'java-library'
-
-ext {
-    sailfishVersion                 = '3.2-SNAPSHOT'
 }
 
 sourceCompatibility = 1.11
```

Comment:
Should be `11` instead of `1.11`. Or we can use a constant from the Gradle `JavaVersion.VERSION_11`